packages/fxa-content-server/app/tests/spec/views/account_recovery_confirm_key.js

```diff
@@ -84,12 +81,12 @@ describe('views/account_recovery_confirm_key', () => {
     });
   });
 
-  it('renders view', () => {
+  it('renders the view with the default serviceName from the relier', () => {
```

Comment:
Looks like this test already exists in [service-mixin](https://github.com/mozilla/fxa/blob/2ddb356924e344e661a841439f349d57cff8e6ef/packages/fxa-content-server/app/tests/spec/views/mixins/service-mixin.js#L68), guess it wouldn't hurt.

Reply:
👍 It does, but we also cover setting a non-default `serviceName` on the relier in [a few](https://github.com/mozilla/fxa/blob/2ddb356924e344e661a841439f349d57cff8e6ef/packages/fxa-content-server/app/tests/spec/views/mixins/service-mixin.js#L65) [other](https://github.com/mozilla/fxa/blob/2ddb356924e344e661a841439f349d57cff8e6ef/packages/fxa-content-server/app/tests/spec/views/reset_password.js#L103) [views](https://github.com/mozilla/fxa/blob/2ddb356924e344e661a841439f349d57cff8e6ef/packages/fxa-content-server/app/tests/spec/views/sign_in_password.js#L87). "Account Settings" is what I see when hitting that screen through the e-mail link and checking for it in a test is what caused me to realize the wrong relier file was being imported to capture the default. Probably more important that we have a test here at all than default vs non-default though just to make sure the fixed issue doesn't reoccur. 🤷‍♀️